templates/Element/FilterBox/filter_box_common.twig

```diff
@@ -101,20 +101,22 @@
 
             <span v-if="filter.name === 'categories'">
                 <label>{{ __('Categories') }}</label>
-
                 {% if schemasByType %}
                     {% for type, typeSchema in schemasByType %}
-                        <category-picker
-                            v-if="queryFilter.filter.type == '{{ type }}'"
-                            :categories="{{ typeSchema.categories|json_encode }}"
-                            :initial-categories="initCategories"
-                            id="category-filter"
-                            form="_filters"
-                            label=""
-                            @change="onCategoryChange"
-                        ></category-picker>
+                        {% if typeSchema.categories %}
```

Comment:
sistemerei l'indentazione qui e in altri punti del file (in particolare dove ci sono degli if)